### PR TITLE
release-notes 19.09: removal of non-LTS kernel attrs

### PR DESCRIPTION
**Pull request by d-goldin**
###### Motivation for this change

Even though the release obviously already happened, I think it might
still make sense to add a short note about the attributes not being
supported any longer (and going forward).

Related to: https://github.com/NixOS/nixpkgs/pull/70202

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @alyssais  @grahamc @samueldr @NeQuissimus

### CODE REVIEW

**Review by NeQuissimus — APPROVED** [review]

File: nixos/doc/manual/release-notes/rl-1909.xml [hunk 1]
```diff
@@ -563,6 +563,15 @@
        earlier version of NixOS.
      </para>
    </listitem>
+   <listitem>
+     <para>
+       <literal>linux_5_1</literal>, <literal>linux_5_2</literal> and <literal>linux_5_3</literal> package
+       attributes have been removed to discourage dependence on specific non-LTS kernel versions in stable
+       NixOS releases. Please use an LTS kernel package, <literal>linux_latest</literal> or
+       <literal>linux_testing</literal>. For more details see
+       <link xlink:href="https://github.com/NixOS/nixpkgs/pull/70202">PR #70202</link>
```
Comment (worldofpeace) [inline nixos/doc/manual/release-notes/rl-1909.xml]: Umm, I really don't feel so great about linking PR's in the release notes for context. Can whatever context that could be added be explained here?

**Comment by worldofpeace** [comment]
This is good to add for sure, I think this was confusing already to a lot of people. Even if it was documented to be this way and it wasn't.

File: nixos/doc/manual/release-notes/rl-1909.xml (hunk 1 above)
Comment (d-goldin) [reply nixos/doc/manual/release-notes/rl-1909.xml]: I somewhat agree, even if this has the most unfiltered information about the whys (like in https://logs.nix.samueldr.com/nixos-dev/2019-10-07#1570476520-1570473549;).

It feels to me like there should be 1-2 paragraphs in the kernel section in the manual about supported kernel versions and pitfalls. But meanwhile, I'll try to see if I can come up with a short half-sentence to replace the PR link without going to deep.

Comment (d-goldin) [reply nixos/doc/manual/release-notes/rl-1909.xml]: @NeQuissimus @worldofpeace: I tried to elaborate a little bit on the intention and improve phrasing. Let me know what you think.

**Comment by d-goldin** [comment]
@NeQuissimus: Thanks! Could you please also take care of throwing this one on master too, so that we don't get inconsistencies with future releases of the manual?

**Comment by worldofpeace** [comment]
Forward ported in a16de399a169dde964d53e570e936def639ebdc2 :smile:

**Pull request merged**